components/profile/mem.rs

```diff
@@ -112,7 +113,17 @@ impl Profiler {
                     })
                     .collect();
                 let _ = sender.send(MemoryReportResult { results });
-                servo_allocator::dump_unmeasured();
+
+                if let Ok(value) = std::env::var("UNTRACKED_LOG_FILE") {
```

Comment:
What should we do if the environment variable is not set? Should we show that to the user and bail out, or silently continue? I am not sure why you would run the allocator feature without showing its output.